mintpy/save_hdfeos5.py

```diff
@@ -42,8 +42,10 @@ def create_parser():
     parser.add_argument('timeseries_file', default='timeseries.h5', help='Timeseries file')
     parser.add_argument('-t', '--template', dest='template_file', help='Template file')
 
-    parser.add_argument('-c', '--coherence', dest='coherence_file', required=True, 
-                        help='Coherence/correlation file, i.e. avgSpatialCoh.h5, temporalCoherence.h5')
+    parser.add_argument('-tc', '--temporal-coherence', dest='temporalcoherence_file', required=True, 
+                        help='Temporal coherence/correlation file, i.e. temporalCoherence.h5')
+    parser.add_argument('-asc', '--average-spatial-coherence', dest='avgspatialcoherence_file', required=True,
+                        help='Average spatial coherence file, i.e. avgSpatialCoh.h5')
```

Comment:
All options with more than one character are with two "-" in mintpy, could you adjust these two also?